survey/src/main/java/com.quickbirdstudios.surveykit/AnswerFormat.kt

```diff
@@ -117,6 +117,34 @@ sealed class AnswerFormat {
         }
     }
 
+    object LocationAnswerFormat : AnswerFormat() {
```

Comment:
As a comment above that:
```kotlin
    /**
     * Following permissions need to be added to the manifest and requested beforehand:
     * <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     * <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     * <uses-permission android:name="android.permission.INTERNET" />
     */
```

Reply:
I moved to api keys to `local.properties` and create methods in Kotlin DSL to run example and test app. Other apps needs to append on their `AndroidManifest.xml`. I explained on the readme.